Automattic/jetpack
# PHPCS: Stats module

I wanted to touch the stats modules for an unrelated issue, but wanted to clean it up before doing so.

#### Changes proposed in this Pull Request:

* PHPCS stats module

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Make sure stats are still collected.
* View the Dashboard in wp-admin and confirm the stats chart looks good.
